docs/commands.rst

```diff
@@ -127,13 +127,28 @@ Specifying 0 as the target version will revert all migrations.
 
         $ phinx rollback -e development -t 0
 
+To rollback all migrations to a specific date then use the ``--date``
+parameter or ``-d`` for short.
+
+.. code-block:: bash
+
+        $ phinx rollback -e development -d 20120103
+        $ phinx rollback -e development -d 20130205120000
+
 If a breakpoint is set, blocking further rollbacks, you can override the
 breakpoint using the ``--force`` parameter or ``-f`` for short.
 
 .. code-block:: bash
 
         $ phinx rollback -e development -t 0 -f
 
+.. note::
+
+        When rolling back, Phinx orders the executed seed migrations using 
```

Comment:
The use of the word "seed" here is probably confusing as migrations and seeds are separate entities.

Seeds are done in a different order and not affected by `version_order`.